libraries/adaptive-expressions/adaptive/expressions/expression.py

```diff
@@ -242,3 +242,47 @@ def try_evaluate(self, state: MemoryInterface, options: Options = None):
         return self.evaluator.try_evaluate(self, state, options)
 
     # TODO: toString
+    def to_string(self):
+        import constant as const
+        builder = ""
+        valid = False
+        expr_type = self.evaluator.expr_type
+        # Special support for memory paths
+        if expr_type == ACCESSOR and len(self.children) >= 1:
+            # pylint: disable=line-too-long
+            if isinstance(self.children[0], const.Constant) and isinstance(const.Constant(self.children[0]).get_value(), str):
```

Comment:
>const.Constant [](start = 75, length = 14)

this is no need since self.children[0] is already Constant type

Reply:
fixed